### DISCUSSION TITLE
Transport submarines from levels 2 to 3 do not have a medical fabricator

### DISCUSSION THREAD

**Discussion by PlasmaUwU** [FakeFishGames/Barotrauma#11121] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

This is very illogical and spoils the balance a bit - after all, why buy an R-29 when there is a Camel?

Even the description of the transport class says that "Transport submarines are equipped to collect and process resources", which is not quite true

Absence of a Medical Fabricator on Berilia
![image](https://user-images.githubusercontent.com/101663742/216103748-178bd2be-c198-4df2-ba93-daa41572266a.png)

Absence of a Medical Fabricator on R-29
![image](https://user-images.githubusercontent.com/101663742/216103951-e024da48-32a2-4d24-8aa7-4073c9ea52c8.png)

### Reproduction steps

1. Open the Submarine editor / start the game
2. Open Berulia or R29 / create them using the console
3. Observe that there is no medical fabricator on them

### Bug prevalence

Happens every now and then

### Version

Faction/endgame test branch

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
Yeah unfortunately due to the small number of vanilla submarines currently in the game, there's just not enough variance to cover the full gamut of capabilities the campaign balance / mechanics expect. Now the enforcement of "tiers" has further fractured the small list of submarines. You can find many other examples not just among transport-class submarines.
